rust/platform/triple_mappings.bzl

```diff
@@ -66,6 +66,41 @@ _SYSTEM_TO_DYLIB_EXT = {
     "unknown": ".wasm",
 }
 
+# See https://github.com/rust-lang/rust/blob/master/src/libstd/build.rs
+_SYSTEM_TO_STDLIB_LINKFLAGS = {
+    # TODO: Right now bazel cc rules does not specify the exact flag setup needed for calling out
+    # TODO: system libs, that is we dont know given a toolchain if it should be, for example,
+    # TODO: `-lxxx` or `/Lxxx` or `xxx.lib` etc.
+    # TODO: We include the flag setup as they are _commonly seen_ on various platforms
+    # TODO: with a cc_rules style override for people doing things like mingw on windows.
+    # TODO: If you are reading this ... sorry! set the env var `BAZEL_RUST_STDLIB_LINKFLAGS` to
+    # TODO: what you need for your specific setup, for example like so
+    # TODO: `BAZEL_RUST_STDLIB_LINKFLAGS="-ladvapi32:-lws2_32:-luserenv"`
```

Comment:
TODO format: just put one TODO(#bug): and then you can do a multiline description. having one TODO per line is hard to read and make me think that this is several things to do.

Reply:
Like this ?

```python
"""TODO(#some_bug)
Right now bazel cc rules does not specify the exact flag setup needed for calling out
system libs, that is we dont know given a toolchain if it should be, for example,
`-lxxx` or `/Lxxx` or `xxx.lib` etc.

We include the flag setup as they are _commonly seen_ on various platforms
with a cc_rules style override for people doing things like mingw on windows.

If you are reading this ... sorry! set the env var `BAZEL_RUST_STDLIB_LINKFLAGS` to
what you need for your specific setup, for example like so
`BAZEL_RUST_STDLIB_LINKFLAGS="-ladvapi32:-lws2_32:-luserenv"`
"""
```
?

Reply:
Yes :) docstring is the best for multiline TODOs.

Reply:
ah sorry I already updated it in keeping with the style that was in the file

Reply:
No worries, using `#` is also fine.